client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -172,12 +172,6 @@ export class CommentNavigation extends Component {
 					) }
 				</NavTabs>
 
-				<CommentNavigationTab className="comment-navigation__actions comment-navigation__open-bulk">
```

Comment:
Another option is to do what we did below with search, and add a boolean prop: useBulkActions, that defaults to false. We can remove the prop when we want to show it always.